prisma/schema.prisma

```diff
@@ -76,28 +76,54 @@ model Org {
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
   users     Member[]
+  roles     Role[]
 }
 
-enum AccessEnum {
```

Comment:
With [this](https://github.com/opencapco/opencap.co/pull/22/files#r1451227203) being said, Lets bring this back